backend/utils/exception_handler.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.response import Response
+
+from rest_framework.views import exception_handler
+from .generic_errors import SlideRequestError, UploadSlidesError
+
+
+def custom_exception_handler(exc, context):
+    response = exception_handler(exc, context)
+
+    if response is not None:
+        response.data['status_code'] = response.status_code
+        return response
+
+    if response is None:
+        response = {
+            "message": str(exc),
+            "status_code": 500,
+        }
+
+        if isinstance(exc, SlideRequestError):
```

Comment:
now this code will be coupled and will change each time we add a new exception type, creating a **BaseException** will reduce all of these exceptions to a single if statement.